client/my-sites/plans/trials/business-trial-expired/index.tsx

```diff
@@ -41,6 +42,21 @@ const BusinessTrialExpired = (): JSX.Element => {
 		siteIsAtomic && selectedSite?.URL
 			? `${ selectedSite.URL }/wp-admin/export.php`
 			: `/export/${ siteSlug }`;
+	const settingsDeleteSiteUrl = `/settings/delete-site/${ siteSlug }`;
+
+	const onDeleteClick = useCallback(
+		( e: React.MouseEvent< HTMLButtonElement > ) => {
+			e.preventDefault();
+
+			recordTracksEvent( 'calypso_plan_trial_expired_page_delete_site', {
+				site_id: siteId,
+				site_slug: siteSlug,
+				trial_type: 'migration',
```

Comment:
Since we're looking for more broad type here, could we change this to `business` instead?